<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SellManForm</class>
 <widget class="QWidget" name="SellManForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>450</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>450</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>700</width>
     <height>450</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>销售单</string>
    </attribute>
    <widget class="QTableView" name="sell_TView">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>641</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="sortSellNum">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>升序</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>降序</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>销售数量排序</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delSell_Btn">
     <property name="geometry">
      <rect>
       <x>534</x>
       <y>10</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>剔除销售表</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sortSELL_Btn">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>排序</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sortSellPrice">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>升序</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>降序</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>销售金额排序</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>交易单</string>
    </attribute>
    <widget class="QTableView" name="trade_TView">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>641</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="delTrade_Btn">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>10</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>剔除交易表</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sortTRADE_Btn">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>排序</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>排序类型</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sortKind">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>员工销售金额</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>会员消费</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="sortType">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>升序</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>降序</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>排序方式</string>
     </property>
    </widget>
    <widget class="QPushButton" name="showAllTrade_Btn">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>显示全部</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
